packages/pharos-site/src/pages/components/component-status/status.json

```
@@ -669,11 +669,11 @@
       "description": ""
     },
     "tests": {
-      "status": "planned",
+      "status": "released",
```

```suggestion
      "status": "planned",
```
This refers to functional tests, which have not been made yet for this component.